### ISSUE TITLE
Demo : how to use on standard (raw) diff urls?

### ISSUE THREAD

**Issue by drehenr** [rtfpessoa/diff2html#340]
### Step -1: Before filling an issue check out troubleshooting section

- Go to [README.md#Troubleshooting](https://github.com/rtfpessoa/diff2html#troubleshooting)
Done, does not help

### Step 0: Describe your environment
Using your own demo instance in the web : https://diff2html.xyz/demo.html 
- OS: **\_**
- diff2html version: **\_**
- Using diff2html directly or using diff2html-ui helper: **\_**
- Extra flags: **\_**

### Step 1: Describe the problem:
Your demo is just great !!!! Many thanks !! But how do I get it to work with URLs that point to a raw unified diff file, like : 
https://raw.githubusercontent.com/thombergs/diffparser/master/src/test/resources/org/wickedsource/diffparser/unified/git.diff ???
I would like to use it for Raw Unififed Diff files like that created by SVN and other tools, not only Github, Gitlab etc.
(The CLI version obviously can handle those files, but I dont get it to work when using -i AND -F option together, and it also always opens a local file in the browser, which obviously prevents it from working in a Groovyscript)

Many thanks in advance !

#### Steps to reproduce:

1. ***
2. ***
3. ***

#### diff example:

```diff
diff --git describe.c
index fabadb8,cc95eb0..4866510
--- a/describe.c
+++ b/describe.c
@@@ -98,20 -98,12 +98,20 @@@
   return (a_date > b_date) ? -1 : (a_date == b_date) ? 0 : 1;
 }
```

#### Observed Results:

- What happened? This could be a description, log output, etc.

#### Expected Results:

- What did you expect to happen?

#### Relevant Code:

```
// TODO(you): code here to reproduce the problem
```

**Comment by rtfpessoa** [comment]
Regarding the demo, for that link it should work. So I am leaving open as a bug.

About the cli, can you post here the command you used that is not working?
Something like `git diff HEAD~5 | diff2html -i stdin -F /tmp/cenas.diff -o stdout` should work without opening the browser.

But after reading the docs seems like `-F` should in fact override the `-o` default. I opened a bug in the cli for that (https://github.com/rtfpessoa/diff2html-cli/issues/108)

**Comment by rtfpessoa** [comment]
Released in version `3.1.14`
